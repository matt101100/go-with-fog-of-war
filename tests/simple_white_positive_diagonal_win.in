place J1
place A1
place A2
place B2
place A3
place C3
place A4
place D4
place A5
place E5
