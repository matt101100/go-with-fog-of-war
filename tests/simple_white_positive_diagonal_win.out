White wins!
J1A1A2B2A3C3A4D4A5E5
Thank you for playing!
